public/ng-views/maps.html

```diff
@@ -268,24 +267,24 @@
       </h6>
 
       <h6 ng-show="targetFeature.alternatives">
-        There are {{ targetFeature.alternatives.length/2 + variants[tabnav].length }} path alternatives along this corridor. {{ variants[tabnav].all | keyLength }} variations, based off of these route path alternatives, have been created this session. They are:
+        There are {{ targetFeature.alternatives.length/2 + variants[tabnav].length }} path alternatives along this corridor. {{ variants[tabnav].all | keyLength }} variations, based off of these route path alternatives, have been created this session. They are: 
```

Comment:
in your revised layout the top two quadrants on the left and right remain, this means there is a clear broken aspect in terms of both the archtecture of the page and in terms of the design. should we resolve this or is this something that is not cared about?